devices/xiaomi.js

```diff
@@ -921,9 +922,14 @@ module.exports = [
                 'without considering right and left sides'),
             exposes.enum('approach_distance', ea.ALL, ['far', 'medium', 'near']).withDescription('The distance at which the ' +
                 'sensor detects approaching'),
-            exposes.numeric('power_outage_count', ea.STATE).withDescription('Number of power outages (since last pairing)')],
+            exposes.enum('motion_sensitivity', ea.ALL, ['low', 'medium', 'high']).withDescription('Different sensitivities ' +
+                'means different static human body recognition rate and response speed of occupied'),
+            exposes.enum('reset_nopresence_status', ea.SET, ['Reset']).withDescription('Reset the status of no presence'),
```

Comment:
Based on the description, I'm not sure what this does, could you explain it further?

Reply:
This function is described in the changelog for the firmware as _"Support resetting nobody. Click to reset the status of nobody."_, and this is how it looks in the Aqara Home application:

![reset_nopresence_status](https://user-images.githubusercontent.com/38351800/157334354-473b4656-fa15-4918-b024-896fb870e642.png)

I also do not quite understand what this function is, but perhaps it is needed to reset the frozen state of the sensor, although I have not encountered this yet.